### DISCUSSION TITLE
Get error when setting APP_ENV=prod

### DISCUSSION THREAD

**Discussion by bpoulsendk** [uvdesk/community-skeleton#665] [Q&A] [answered]
Hello everybody.

So I've managed to get Uvdesk running and all features work.

When I go to set APP_ENV to prod and want to clear cache I get a ton of errors like this:

`deprecation.INFO: User Deprecated: Since symfony/security-bundle 5.3: The child node "encoders" at path "security" is deprecated, use "password_hashers" instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Since symfony/security-bundle 5.3: The child node \"encoders\" at path \"security\" is deprecated, use \"password_hashers\" instead. at /var/www/public_html/helpdesk/vendor/symfony/security-bundle/DependencyInjection/MainConfiguration.php:73)"} []`

I've tried looking into the files mentioned but can't seem to figure this out.

**Comment by komal-sh-27** [comment, marked as answer]
@bpoulsendk 

>When I go to set APP_ENV to prod and want to clear cache I get a ton of errors like this:

We are informing to you, these are only showing a depreciation messages not an error. These depreciation message shows only on a prod mode. 
If you are facing any other issue then let us know.

**Comment by PeopleInside** [comment]
This deprecation message are for developers to know in future will need to update the component. You can ignore it :)

**Reply by bpoulsendk** [reply]
Well maybe.. But I get

500 - Internal server error
Our system has goofed up for a while, but good part is it won't last long

if I reload the page after i set it to prod.

And trying to log in i get "Authentication request could not be processed due to a system problem.".

So I have to set it back to dev to even be able to log in or see the dashboard.

**Reply by PeopleInside** [reply]
What version of PHP are you using?
What the server error log say?

**Comment by bpoulsendk** [comment]
FYI: The 500 internal error disappeared after a little while but the authentication request could not be processed due to a system problem is still there.

**Comment by komal-sh-27** [comment]
@bpoulsendk 

Unfortunately, we are not able to reproduce this issue. If you have installed your project on the server then you can share with us your server details on our support panel. You can create a ticket on our support panel from [here](https://support.uvdesk.com/en/customer/create-ticket/).

So that we can check this issue.

**Comment by bpoulsendk** [comment]
Thank you very much for your answers.. I'll try and see if a clean install will help :)

**Comment by floviolleau** [comment]
Hi,

I have the same issue shown here.

Any clues?

Do we need to perform other post-install steps (like setting APP_ENV=prod?

Thanks for your help

**Reply by PeopleInside** [reply]
Currently Uvdesk Is no more getting updated and support.

**Comment by alex23232-tech** [comment]
Looking to streamline your hiring process? Check out our [recruitment page](https://hrbs.com.pk/recruitment-headhunting/) for innovative solutions and tips to attract top talent.
